markdown/org/docs/designs/jane/needs/en.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Jane Shift: What you need"
+---
+
+To determine how much fabric you need for Jane you take:
+- Twice the length of your shift + the length of your sleeves
+- If your sleeves and gussets can fit alongside the maximum width of your shift, then twice the length of the shift will be enough.
```

Comment:
```suggestion
- Twice the length of your body pattern piece + the length of your sleeve pattern piece.
- If your sleeves and gussets can fit alongside the maximum width of your body pattern piece, then twice the length of the body pattern piece will be enough.
```
It's not call shift on the design it may cause confusion